<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profiled_designer</class>
 <widget class="QMainWindow" name="profiled_designer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>651</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>ProfiLED Designer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/ProfiLED.bmp</normaloff>:/icon/ProfiLED.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="globalSettings">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>179</width>
         <height>516</height>
        </rect>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="label">
        <string>Global</string>
       </attribute>
       <widget class="QSpinBox" name="loop_duration">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>48</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Loop Duration(s)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="play_button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pause_button">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
       <widget class="QPushButton" name="create_bin">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Bin</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ledSettings">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>179</width>
         <height>516</height>
        </rect>
       </property>
       <attribute name="label">
        <string>LED</string>
       </attribute>
       <widget class="QLineEdit" name="solid_color_text">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="solid_color">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Solid Color</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="pattern">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Pattern</string>
        </property>
       </widget>
       <widget class="QListWidget" name="pattern_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="color_select">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>250</y>
          <width>26</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_pattern">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="remove_pattern">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>LED ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="led_id">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="total_time">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QToolButton" name="start_color_select">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>360</y>
          <width>26</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="start_color">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Color</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>End Color</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="end_color">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="end_color_select">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>410</y>
          <width>26</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Total Time (10ms)</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="offset_time">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>180</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Offset Time (10ms)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Pattern Mid (%)</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="pattern_mid">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>310</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>450</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete LED</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>480</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Duplicate LED</string>
        </property>
       </widget>
       <zorder>solid_color_text</zorder>
       <zorder>solid_color</zorder>
       <zorder>pattern</zorder>
       <zorder>pattern_list</zorder>
       <zorder>color_select</zorder>
       <zorder>label_2</zorder>
       <zorder>led_id</zorder>
       <zorder>total_time</zorder>
       <zorder>start_color_select</zorder>
       <zorder>start_color</zorder>
       <zorder>label_3</zorder>
       <zorder>label_4</zorder>
       <zorder>end_color</zorder>
       <zorder>end_color_select</zorder>
       <zorder>label_5</zorder>
       <zorder>offset_time</zorder>
       <zorder>label_6</zorder>
       <zorder>label_7</zorder>
       <zorder>pattern_mid</zorder>
       <zorder>remove_pattern</zorder>
       <zorder>add_pattern</zorder>
       <zorder>pushButton</zorder>
       <zorder>pushButton_2</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
